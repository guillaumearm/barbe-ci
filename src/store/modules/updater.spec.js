const updater = require('./updater');

describe('store/updater', () => {
  const unknownAction = updater({ type: 'UNKNOWN' });
  const initialState = {
    ...unknownAction(undefined),
    credentials: { token: 'abcdef' },
    serverConfiguration: { VERBOSE: true },
  };
  it('returns initial state', () => {
    expect(unknownAction(undefined)).toMatchSnapshot();
  });
  it('does not update state on unknown action', () => {
    expect(unknownAction(initialState)).toEqual(initialState);
  });
});
