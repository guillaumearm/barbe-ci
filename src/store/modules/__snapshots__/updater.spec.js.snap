// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store/updater returns initial state 1`] = `
Object {
  "credentials": Object {},
  "db": Object {
    "commits": Object {},
    "repositories": Object {},
    "tokens": Object {},
    "users": Object {},
  },
  "logs": Array [],
  "serverConfiguration": Object {},
}
`;
